config/polyfills.js

```diff
@@ -23,14 +21,14 @@ Object.assign = require('object-assign');
 // Removes:
 // Warning: React depends on requestAnimationFrame. Make sure that you load a
 // polyfill in older browsers.
-global.requestAnimationFrame = function(callback) {
+global.requestAnimationFrame = function (callback) {
   setTimeout(callback, 0);
 };
 
 /* eslint-disable */
 // Ensuring polyfills for include available to IE
 if (!String.prototype.includes) {
-  String.prototype.includes = function(search, start) {
+  String.prototype.includes = function (search, start) {
```

Comment:
can get rid of these for the same reason.

Reply:
Removed polyfills in https://github.com/cfpb/ccdb5-ui/pull/426/commits/606db613e958c3c828f28d71d28a0dbf1d4b0093

Reply:
Note this also gets rid of the promise rejection tracking, which doesn't seem to be in explorer.